src/components/RestCourses/Tags.js

```diff
@@ -0,0 +1,151 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import 'antd/dist/antd.css';
+import { Tag, Input, Tooltip } from 'antd';
+import { PlusOutlined } from '@ant-design/icons';
+
+export default class Tags extends React.Component {
```

Comment:
woof... why the class component? back in the stone ages

Reply:
jkjk the class component is totally fine. just surprised to see it

Reply:
100% switching it over to a functional component. The only reason it comes in a class component is because the ant d docs suggest them like this